internal/controller/controllers/local_cluster_import_controller_test.go

```diff
@@ -29,7 +30,9 @@ import (
 	fakeclient "sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
-const localClusterName = "local-cluster"
+const localClusterName = "arbitrary-cluster-name"
+
+var localClusterID = uuid.NewString()
```

Comment:
Any reason we can't put this in the per-test-lifecycle? (i.e. generate a new one in `BeforeEach`)?